src/pages/experiments/[experimentId]/plots-and-tables/PlotsTablesContainer.jsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
```

Comment:
having this in pages seems to add a new route `/plots-and-tables/PlotsTablesContainer` which I assume it's not what we want. Shouldn't this be placed in `components` ?

Reply:
Yep, my bad. Moved under `components`.